packages/playground/rescript.json

```diff
@@ -5,11 +5,6 @@
     "dir": "src",
     "subdirs": true
   },
-  "package-specs": {
-    "module": "esmodule",
-    "in-source": false,
```

Comment:
@tsnobip do you have any idea why the playground has "in-source": false?

Reply:
no idea, that hasn't changed in years!